crates/wit-bindgen-wasmcloud-provider-binary/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "wit-bindgen-wasmcloud-provider-binary"
+version = "0.1.0"
+authors.workspace = true
+categories.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+publish = false
+description = """
+Bindgen macro for binary-based wasmCloud capability providers written in Rust
+"""
+
+[lib]
+proc-macro = true
+
+[features]
+default = []
+debug = []
```

Comment:
What does this do?

Reply:
old code here, this feature was removed, will remove